chart/kubeapps/values.schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "frontend": {
+      "type": "object",
+      "title": "Frontend configuration",
+      "form": true,
+      "properties": {
+        "replicaCount": {
+          "type": "integer",
+          "title": "Number of replicas",
+          "form": true
+        }
+      }
+    },
+    "apprepository": {
+      "type": "object",
+      "title": "Chart repositories configuration",
+      "form": true,
+      "properties": {
+        "initialRepos": {
+          "type": "array",
+          "title": "Initial chart repositories",
+          "description": "Initial chart repositories to configure on Kubeapps",
+          "form": true
+        }
```

Comment:
I would remove this parameter for two reasons:
 - As you said we don't have support for arrays yet so the representation is confusing.
 - This form would be shown only when upgrading (unless you are deploying a second instance of Kubeapps). The `initialRepos` are only meant to be set the first time you install the chart (that's why it's called `initial*`). Changing this parameter when upgrading will have no effect since after installation, repositories are managed using the Kubeapps UI.